packages/core/src/plugins/rspackProfile.ts

```diff
@@ -88,6 +86,10 @@ export const pluginRspackProfile = (): RsbuildPlugin => ({
     api.onBeforeStartDevServer(onStart);
 
     api.onAfterBuild(async ({ stats }) => {
+      const profileDir = path.join(api.context.distPath, profileDirName);
+
+      const loggingFilePath = path.join(profileDir, 'logging.json');
```

Comment:
The join calls can be merged:

```suggestion
      const loggingFilePath = path.join(api.context.distPath, profileDirName , 'logging.json');
```